.github/workflows/ci-core.yml

```diff
@@ -240,17 +240,10 @@ jobs:
         id: run-tests
         env:
           OUTPUT_FILE: ./output.txt
-          USE_TEE: false
+          USE_TEE: ${{ matrix.type.printResults }}
```

Comment:
`USE_TEE` is actually not supported anymore. However, it will use `tee` if `$DEBUG` is `true`.

So we can set debug to true when printResults is true, and it's not a scheduled event:

```suggestion
          DEBUG: ${{ matrix.type.printResults == 'true' && github.event_name != 'schedule'  }}
```

Could also use the filtered test results when it is a scheduled run, as that uses JSON.

```
      - name: Print Filtered Test Results
        if: ${{ failure() && needs.filter.outputs.should-run-ci-core == 'true' && steps.run-tests.conclusion == 'failure' && github.event_name == 'schedule' }}
        run: |
          if [[ "${{ matrix.type.printResults }}"  == "true" ]]; then
            cat output.txt | gotestloghelper -ci
          fi
```

Reply:
I agree, the pipeline itself and these scripts are too complex.

I'm heads down on the test binary hashing solution, which will likely blow up this pipeline so not sure if it's worth investing a lot of time to simplify the existing one.

Reply:
@erikburt maybe this is good enough for now? https://github.com/smartcontractkit/chainlink/pull/15395